html/browsers/windows/noreferrer-window-name.html

```diff
@@ -0,0 +1,44 @@
+<!doctype html>
+<title>rel=noreferrer and reuse of names</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<div id="log"></div>
+<script>
+  async_test(function(t) {
```

Comment:
If we're sure we're not stomping on anyone's toes, then we should probably `clear()` at the beginning and end of the test?